### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ymigval**
For those who are experimenting with things quickly, we may encounter issues like the one I experienced. I was using a model that has a custom date field that I wanted to use as the date field, but I didn't see a solution in the README. So, I decided to inspect the code to figure out how to address this problem. I have added an additional note to the README so that this can be taken into account at first glance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,3 +120,6 @@ Please review [our security policy](../../security/policy) on how to report secu
 ## License
 
 The MIT License (MIT). Please see [License File](LICENSE.md) for more information.
+```
```
Comment (Larsklopstra) [inline README.md:123]: Hmm, this looks like it should not be here?

**Review by Larsklopstra — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ymigval) [reply README.md]: That's right, I slipped that in there but it's already solved

Comment (Larsklopstra) [reply README.md]: > This addition clarifies the usage of a custom date column in your Laravel Trend package.

Hmm, why's it below the license? Maybe just remove it :)

Comment (ymigval) [reply README.md]: That's right, line deleted

**Pull request merged**